CHANGELOG.md

```diff
@@ -1,16 +1,69 @@
 ## 0.17.11 (Unreleased)
 
+### Major Changes
+
+#### Secrets and Pluggable Encryption
+
+- The Pulumi engine and Python and NodeJS SDKs now have support for tracking values as "secret" to ensure they are
+  encrypted when being persisted in a state file.
+
+  Any existing value may be turned into a secret by calling `pulumi.secret(<value>)` (NodeJS) or
+  `Output.secret(<value>`) (Python).  In both cases, the returned value is an Output which may be passed around
+  like any other.  If this value flows into a resource, the plaintext will not be stored in the state file, but instead
+  It will be encrypted, just like values added to config with `pulumi config set --secret`.
+
+  If an output which has been marked as secret is combiend with other outputs (either via `all` or `apply`) the
+  resulting output value will also be treated as a secret.
```

Comment:
I don’t think we need to document the feature in the changelog. Let’s keep this high level, link to PR/issue, and make sure to update docs with all of this additional info ASAP (and update the changelog if needed after that is done).